src/apache/conf/ssl.conf

```diff
@@ -161,3 +161,17 @@ SSLRandomSeed connect file:/dev/urandom 512
         Header always set Strict-Transport-Security "max-age=63072000; includeSubdomains;"
     </IfDefine>
 </VirtualHost>
+
+#########################
+#        Snapweb        #
+#########################
+Listen 4201
+<VirtualHost *:4201>
```

Comment:
This also means one can only use snapweb from within nextcloud if SSL is enabled, right? Isn't that a little odd?